packages/admin-app/package.json

```diff
@@ -9,10 +9,18 @@
         "@types/jest": "^26.0.15",
         "@types/node": "^12.0.0",
         "@types/react": "^17.0.0",
+        "@types/react-router-dom": "^5.1.7",
         "@types/react-dom": "^17.0.0",
+        "@typescript-eslint/eslint-plugin": "^4.22.0",
```

Comment:
These eslint pieces will need to be removed. See #69 . They should exist at top level.

Reply:
okay got it